root/commons/src/main/java/at/fhv/itb5c/commons/property/PropertyManager.java

```diff
@@ -1,16 +1,39 @@
 package at.fhv.itb5c.commons.property;
 
+import java.io.IOException;
 import java.io.InputStream;
+import java.util.Properties;
 
-public class PropertyManager {
+import at.fhv.itb5c.logging.ILogger;
+
+public class PropertyManager implements ILogger {
+	private PropertyManager _instance;
+	private Properties _properties;
 	private InputStream _inputStream;
 	private final String _fileName = "config.properties";
 
-	public PropertyManager() {
-		// TODO Auto-generated constructor stub
+	private PropertyManager() {
+		_inputStream = getClass().getClassLoader().getResourceAsStream(_fileName);
+
+		try {
+			if (_inputStream != null) {
+				_properties.load(_inputStream);
+			} else {
+				// TODO create file if doesnt exist				
```

Comment:
I think this is not enough. What if the file is empty? We need a way to guarantee that all properties have a fallback value.